### PR TITLE
Fetch min/max range for current time step on parameter hover

### PR DESCRIPTION
**Pull request by bnmajor**
As a first pass a retrieving and displaying the minimum and maximum values for a parameter at the current time step this is a bit too slow. There is a delay in fetching the ranges, so moving the mouse around with any speed causes values to blink in and out after the hover action has finished.

hover-range

Currently to get the range we:
- fetch the item
- fetch the file information (to get the id)
- download the file
- extract the range

But this is too slow and would need to be approached from a different angle if this ends up being a desired feature.

### CODE REVIEW

**Comment by cjh1** [comment]
@bnmajor I don't thing its that bad. However, I think we need to added check to stop the tooltip showing up if the mouse have already left the row when the data return, otherwise you get trail effect of tooltips.

File: client/src/App.vue [hunk 1]
```diff
@@ -158,31 +157,45 @@ export default {
       }
     },
 
-    hover(event) {
+    hoverOut() {
       this.range = '';
```
Comment (cjh1) [inline client/src/App.vue]: If we can we should cancel the request made by the `hoverIn` is possible

Comment (cjh1) [reply client/src/App.vue]: we might also want debounce the entry/exit.

**Comment by cjh1** [comment]
This is look much better

**Comment by cjh1** [comment]
I am getting the following error with this branch: 

```

Vue packages version mismatch:

- vue@2.6.11 (/home/cjh/work/source/adios-dashboard/client/node_modules/vue/dist/vue.runtime.common.js)
- vue-template-compiler@2.6.10 (/home/cjh/work/source/adios-dashboard/client/node_modules/vue-template-compiler/package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the latest.

    at Object.<anonymous> (/home/cjh/work/source/adios-dashboard/client/node_modules/vue-template-compiler/index.js:10:9)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/cjh/work/source/adios-dashboard/client/node_modules/vuetify-loader/lib/loader.js:3:18)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at loadLoader (/home/cjh/work/source/adios-dashboard/client/node_modules/loader-runner/lib/loadLoader.js:18:17)
    at iteratePitchingLoaders (/home/cjh/work/source/adios-dashboard/client/node_modules/loader-runner/lib/LoaderRunner.js:169:2)

 @ ./node_modules/@girder/components/src/components/Authentication/index.js 1:0-50 3:15-29
 @ ./node_modules/@girder/components/src/components/index.js
 @ ./node_modules/@girder/components/src/index.js
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://10.50.58.179:8080/sockjs-node (webpack)/hot/dev-server.js ./src/main.js
```

Does if need rebasing on current master?

**Comment by cjh1** [comment]
The is better, thanks

**Comment by bnmajor** [comment]
Yes it did need rebasing, should be fixed now.

Also hovering now makes only 5-6% of the calls it was making previously. Big improvement.

**Pull request merged**